tests/ParserTest.elm

```diff
@@ -912,6 +912,28 @@ expr =
                   )
                 ]
               )
+            , ( "cons"
+              , [ ( "simple case"
+                  , "1 :: []"
```

Comment:
Let's add tests for whether it works with different amount of spaces (none, multiple). Don't do newlines for now

Reply:
This is also done! 🎉